capstone-app/src/app/modules/material/material.module.ts

```diff
@@ -7,12 +7,14 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { MatTabsModule } from '@angular/material/tabs';
 import { MatToolbarModule } from '@angular/material/toolbar';
-import { MatListModule } from '@angular/material/list';
 import { MatDividerModule } from '@angular/material/divider';
 import { MatProgressBarModule } from '@angular/material/progress-bar';
-import { MatExpansionModule } from '@angular/material/expansion';
 import { MatCardModule } from '@angular/material/card';
 import { MatNativeDateModule } from '@angular/material/core';
+import { MatDialogModule } from '@angular/material/dialog';
```

Comment:
If we're importing MatDialogModule here, then we shouldn't need to import it in app.component.ts

Reply:
we need to import the MatDialog to the app.component.ts because we use "public dialog: MatDialog" in our constructor and we use this public dialog to open our dialog component

Reply:
removing it also causes an error